### PR TITLE
Remove FrontPage Index Pages (non-variant services) - WSTEAMA-382

### PR DESCRIPTION
**Pull request by pvaliani**
Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAMA-382 

Overall changes
======

- Removes `index.json` files for all `frontPage` services except `serbian, zhongwen, uzbek` and `urdu` 
- Retains `urdu` as is a RTL service needed for general testing
- Updates snapshots to reflect changes to `serbian` variants due to need to enrich data for current test implementation
- Fixes an issue where `mostRead` was not rendering on `storybook` due to the `mostReadEndPointOverride` not being passed in to the `mostRead` renderer
- Updates unit and integration tests 

Code changes
======

- As above

### CODE REVIEW

**Comment by karinathomasbbc** [comment]
Hope it's OK for me to snoop on this: I was just checking the FrontPage stories, and noticed that the MostRead component is missing from the Serbian Cyr & Lat pages - https://remove-old-frontpage-index-files-wste--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/pages-front-page--serbian-latin

Could you have a look? We'll need these to verify that we have not 💥 anything with our new Most Read component

**Comment by pvaliani** [comment]
> Hope it's OK for me to snoop on this: I was just checking the FrontPage stories, and noticed that the MostRead component is missing from the Serbian Cyr & Lat pages - https://remove-old-frontpage-index-files-wste--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/pages-front-page--serbian-latin
> 
> Could you have a look? We'll need these to verify that we have not 💥 anything with our new Most Read component

We noticed that the `mostReadEndPointOverride` was not being passed in to `renderMostRead` which was causing the path to default to the live/test (production) URL for the variant index which was a 404. It meant that the result evaluated to `undefined` which is why most read wasn't rendering on stories.

https://github.com/bbc/simorgh/blob/696bee3b117f5321d16c14fe0a3b773a94b5dce8/src/app/pages/FrontPage/FrontPage.jsx#L114

By introducing this change, they're now visible 🤘

File: src/app/pages/FrontPage/FrontPage.jsx [hunk 1]
```diff
@@ -110,7 +110,8 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
         <IndexPageContainer>
           {groups.map((group, index) => (
             <Fragment key={group.title}>
-              {group.type === 'useful-links' && renderMostRead()}
+              {group.type === 'useful-links' &&
+                renderMostRead(mostReadEndpointOverride)}
```
Comment (karinathomasbbc) [inline src/app/pages/FrontPage/FrontPage.jsx:114]: Nice! Thanks for fixing this! I am assuming that it doesn't affect the rendering of the most read on the page (i.e. not via storybook?)

**Review by karinathomasbbc — APPROVED** [review]
LOVE IT - great to see so many deletions. Sorry it was such a pain, but definitely worth it!

File: src/app/pages/FrontPage/FrontPage.jsx (hunk 1 above)
Comment (pvaliani) [reply src/app/pages/FrontPage/FrontPage.jsx:114]: Me and @alex-magana checked this as best we could manually and should be no issues 🤞

**Review by amoore108 — APPROVED** [review]

**Review by alex-magana — APPROVED** [review]
LGTM 🚀

**Pull request merged**